nestkernel/conn_builder.h

```diff
@@ -240,6 +246,12 @@ class AllToAllBuilder : public ConnBuilder
   {
   }
 
+  bool
+  is_symmetric() const
+  {
+    return true;
+  }
```

Comment:
I think this is not always true. If all-to-all has a randomized synapse parameter, it is no longer automatically symmetric. We will still have a backward connection for each forward connection, but their properties will differ.